drivers/ethernet/eth_xlnx_gem.c

```diff
@@ -90,51 +96,18 @@ DT_INST_FOREACH_STATUS_OKAY(ETH_XLNX_GEM_INITIALIZE)
 
 /**
  * @brief GEM device initialization function
- * Initializes the GEM itself, the DMA memory area used by the GEM and,
- * if enabled, an associated PHY attached to the GEM's MDIO interface.
+ * Initializes the respective GEM controller instance itself and its
+ * associated DMA area.
  *
  * @param dev Pointer to the device data
  * @retval 0 if the device initialization completed successfully
  */
 static int eth_xlnx_gem_dev_init(const struct device *dev)
 {
 	const struct eth_xlnx_gem_dev_cfg *dev_conf = dev->config;
```

Comment:
__maybe_unused  when you removed if (dev_conf->init_phy) { below? 

.../zephyr-workspace/zephyr/drivers/ethernet/eth_xlnx_gem.c:109:44: warning: unused variable 'dev_conf' [-Wunused-variable]
  109 |         const struct eth_xlnx_gem_dev_cfg *dev_conf = dev->config;
      |                                            ^~~~~~~~

Reply:
done, added an attribute unused marker to suppress the warning